app/controllers/stability_controller.rb

```diff
@@ -23,6 +25,14 @@ def select_all_names
       sort
   end
 
+  def select_all_plugins
+    Bacon.where("action_name LIKE :prefix", prefix: "fastlane-plugin-%").
```

Comment:
Since there is no variable substitution to do, I think you can just say:

`Bacon.where("action_name LIKE fastlane-plugin-%").`